include/aspect/newton.h

```diff
@@ -156,6 +156,7 @@ namespace aspect
        */
       double              nonlinear_switch_tolerance;
 
+      bool                use_Eisenstat_Walker_method_for_Picard;
```

Comment:
please rename to `use_Eisenstat_Walker_method_for_Picard_iterations` to be consistent with the other parameters